src/admin-views/widgets/components/radio.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Admin View: Widget Radio Component
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/radio.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var string       $label   Label for the radio group.
+ * @var string       $value   Value for the radio.
+ * @var string       $name    Name attribute for the radio.
+ * @var array<array> $options Data for the individual radio buttons.
+ */
+
+if ( ! empty( $label ) ) : ?>
+	<p><?php echo esc_html( $label ); ?></p>
+	<?php
+endif;
+
+$i = 1;
+
+foreach ( $options as $option ) :
+	$radio_id = $id . $i;
+	?>
+	<p class="tribe-widget-radio tribe-common-form-control-radio">
```

Comment:
The enclosed input and label is not a paragraph, it should probably be wrapped in a div instead.

Reply:
OK, should we change that for checkboxes as well?
https://github.com/moderntribe/the-events-calendar/blob/release/B20.11/src/admin-views/widgets/components/checkbox.php#L21